cdap-tms/src/main/java/co/cask/cdap/messaging/store/cache/CachingMessageTable.java

```diff
@@ -64,7 +65,7 @@
                       MessageTableCacheProvider cacheProvider, TimeProvider timeProvider) {
     // Half the tx pruning grace period to be the grace period for scanning the message cache.
     // This is to make sure we won't scan for cached entries that might be pruned.
-    this.gracePeriod = cConf.getLong(PRUNE_GRACE_PERIOD) / 2;
+    this.gracePeriod = cConf.getLong(PRUNE_GRACE_PERIOD, TimeUnit.HOURS.toSeconds(24)) / 2;
```

Comment:
this default value should be in cdap-defaults, or at least in Constants. Not inline in the code.

Reply:
Good point. I'll just use `TxConstants.TransactionPruning.DEFAULT_PRUNE_GRACE_PERIOD`